ui/app/components/console/ui-panel.js

```diff
@@ -8,22 +9,38 @@ import {
   executeUICommand,
 } from 'vault/lib/console-helpers';
 
-const { inject, computed } = Ember;
+const { inject, computed, getOwner, run } = Ember;
 
 export default Ember.Component.extend({
   classNames: 'console-ui-panel-scroller',
   classNameBindings: ['isFullscreen:fullscreen'],
   isFullscreen: false,
   console: inject.service(),
+  router: inject.service(),
   inputValue: null,
   log: computed.alias('console.log'),
 
+  didReceiveAttrs() {
+    let val = this.get('inputValue');
+    let oldVal = this.get('oldInputValue');
+    this.set('valChanged', val !== oldVal);
+    this.set('oldInputValue', val);
+  },
+
+  didRender() {
+    if (this.get('valChanged')) {
+      // make sure we're scrolled to the bottom;
+      this.scrollToBottom();
+    }
+  },
+
   logAndOutput(command, logContent) {
-    this.set('inputValue', '');
     this.get('console').logAndOutput(command, logContent);
+    run.next(()=> this.scrollToBottom());
```

Comment:
I think it's practically equivalent, but semantically it might be clearer to use the `afterRender` queue here instead of `run.next`.